database/migrations/2016_01_29_090810_add_description_to_achievements.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+
+class AddDescriptionToAchievements extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('osu_achievements', function ($table) {
+            $table->text('description')->after('name');
```

Comment:
needs a default of '' or nullable + default NULL else i think this will fail when applying (has no default to apply to existing entries.